test_unstructured_ingest/python/test-ingest-astra-output.py

```diff
@@ -0,0 +1,50 @@
+import click
+import random
+
+from astrapy.db import AstraDB
+
+@click.command()
+@click.option("--token", type=str)
+@click.option("--api-endpoint", type=str)
+@click.option("--collection-name", type=str, default="collection_test")
+@click.option("--embedding-dimension", type=int, default=384)
+def run_check(token, api_endpoint, collection_name, embedding_dimension):
+    print(f"Checking contents of Astra DB collection: {collection_name}")
+
+    # Initialize our vector db
+    astra_db = AstraDB(token=token, api_endpoint=api_endpoint)
+    astra_db_collection = astra_db.collection(collection_name)
+
+    # Tally up the embeddings
+    docs_count = astra_db_collection.count_documents()
+    number_of_embeddings = docs_count["status"]["count"] 
+
+    # Print the results
+    expected_embeddings = 1
```

Comment:
change from 1 to 3 since the chunking produces a document with 3 parts. Which then creates 3 embeddings.